templates/default/tests/markdownFeatures.spec.js

```diff
@@ -96,9 +96,9 @@ test.describe.parallel("commonMark", () => {
   });
 
   test("blockquote", async ({ page }) => {
-    await expect(page.locator("#blockquote > blockquote > p")).toContainText(
-      "I am a block quote."
-    );
+    await expect(
+      page.locator("#blockquote > div > blockquote > p")
+    ).toContainText("I am a block quote.");
   });
 
   // test("lists", async ({ page }) => {
```

Comment:
Let's add some proper tests related to callouts support, e.g. test if the proper icon is displayed depending on the type of the callout (not for all 12, let's say for 2) and test if callout headings are not wrapped in anchor tags.